src/pages/scheduler/components/share/__tests__/ShareTimetableModal.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+
+import { SavedCourse } from 'lib/hooks/useSavedCourses';
+
+import ShareTimetableModal from '../ShareTimetableModal';
+
+const courses: SavedCourse[] = [
+  { subject: 'CSC', code: '111', pid: '1', term: '202109', lecture: 'A01' },
+  { subject: 'MATH', code: '100', pid: '2', term: '202109', lab: 'B01' },
+  { subject: 'PHYS', code: '110', pid: '3', term: '202109', tutorial: 'T01' },
+  { subject: 'ENGR', code: '130', pid: '4', term: '202109', lecture: 'A22', lab: 'B01' },
+];
+
+describe('ShareTimetableModal', () => {
+  describe('when sending params in', () => {
+    it('should display the correct term in readable format', () => {
+      render(
+        <ShareTimetableModal
+          term={'202109'}
+          loading={false}
+          onClose={() => {}}
```

Comment:
you could write a test that verifies that `onClose` is passed in. It would be similar to how I test this [mock function is being called ](https://github.com/VikeLabs/courseup/blob/main/src/lib/utils/jest/index.tsx#L11) in the sense that you create a mock function, pass it in, and use Jest to see if it's called. You would also need to use `userEvent` in a similar way to "click" the button